### PR TITLE
Feature/debug trace updates

### PR DESCRIPTION
**Pull request by conor2e**
## Description
Polygon and Avalanche APIs and fixed typos in others.

### CODE REVIEW

**Review by ginstagram — APPROVED** [review]
LGTM

**Review by inoob26 — APPROVED** [review]

File: docs/operations/avalanche/debug-and-trace-apis.md [hunk 1]
```diff
@@ -8,7 +8,12 @@ meta:
 
 # Debug and trace APIs
 
-A dedicated Avalanche full or archive node has the `debug_*` API methods enabled. For the full list of the available debug and trace API methods, see [Debug namespace](https://geth.ethereum.org/docs/rpc/ns-debug).
+You can deploy an [elastic](/glossary/elastic-node) Avalanche archive node with the enabled debug and trace APIs starting from the <a href="https://chainstack.com/pricing/" target="_blank">Business plan</a> as the Geth [client implementation](/operations/ethereum/clients).
```
Comment (easeev) [inline docs/operations/avalanche/debug-and-trace-apis.md:11]: How come Avalanche node runs as a Ethereum client implementation and not Avalanche client implementation?

Comment (conor2e) [reply docs/operations/avalanche/debug-and-trace-apis.md:11]: Hi @easeev. I understood that the Avalanche client (AvalancheGo) has the same details as the Geth implementation, so linked to the exisiting page. I'll create a new task to provide more detailed info on the Avalanche chains and clients.

File: docs/operations/polygon/debug-and-trace-apis.md [hunk 2]
```diff
@@ -8,7 +8,14 @@ meta:
 
 # Debug and trace APIs
 
-A dedicated Polygon full or archive node has the `debug_*` API methods enabled. For the full list of the available debug and trace API methods, see [Debug namespace](https://geth.ethereum.org/docs/rpc/ns-debug).
+You can deploy an [elastic](/glossary/elastic-node) Polygon archive node with the enabled debug and trace APIs starting from the <a href="https://chainstack.com/pricing/" target="_blank">Business plan</a> as the Erigon [client implementation](/operations/ethereum/clients).
```
[the remark this answers is not in the document]
Comment (conor2e) [reply docs/operations/polygon/debug-and-trace-apis.md:11]: Similar to the Avalanche content, I understood that the Polygon client runs on the Erigon implementaiton, so linked to the exisiting page. I'll also get started on adding more detail about Polygon frameworks and clients to the docs.

File: docs/operations/avalanche/debug-and-trace-apis.md (hunk 1 above)
Comment (conor2e) [reply docs/operations/avalanche/debug-and-trace-apis.md:11]: I agree that I should clarify and add more information, and will correct this now. Thanks.

File: docs/operations/polygon/debug-and-trace-apis.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (conor2e) [reply docs/operations/polygon/debug-and-trace-apis.md:11]: I just checked and the client used when creating the elastic archive node with debug and trace enabled is the Erigon client, so I'll leave that unchanged for now:  
polygon_debug

File: docs/operations/avalanche/debug-and-trace-apis.md (hunk 1 above)
Comment (conor2e) [reply docs/operations/avalanche/debug-and-trace-apis.md:11]: Due to the workflow changes in this project yesterday, I won't unmerge/repopen this PR and have created new one for new updates, including this. https://github.com/chainstack/docs/pull/558

**Pull request merged**